ansible/roles/ocp4-workload-mlops/tasks/install-gogs.yaml

```diff
@@ -31,8 +31,8 @@
     validate_certs: false
   register: result
   until: result.status == 200
-  retries: 50
-  delay: 10
+  retries: 400
```

Comment:
Less retries and longer delay will be better vs hammering the API every 2 seconds for no reason.